src/Geosuggest.jsx

```diff
@@ -20,7 +20,6 @@ const Geosuggest = React.createClass({
       bounds: null,
       country: null,
       types: null,
-      googleMaps: google && google.maps,
```

Comment:
Removing this is a breaking change – People should still be able to pass in Google Maps by themselfs. Maybe make it null per default.